feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go

```diff
@@ -217,6 +229,15 @@ func (tc *testCase) configureDUT(t *testing.T) {
 		lacpPath.Replace(t, lacp)
 	}
 
+	//Unset openconfig-lacp:lacp config "Lacp_LacpActivityType_UNSET"
+	if tc.lagType == lagTypeSTATIC {
+		lacp := &telemetry.Lacp_Interface{Name: ygot.String(tc.aggID)}
+		lacp.LacpMode = telemetry.Lacp_LacpActivityType_UNSET
+		lacpPath := d.Lacp().Interface(tc.aggID)
+		fptest.LogYgot(t, "LACP", lacpPath, lacp)
+		lacpPath.Replace(t, lacp)
+	}
+
```

Comment:
Can we merge the two cases so the difference is more obvious?

```
lacp := &telemetry.Lacp_Interface{Name: ygot.String(tc.aggID)}
if tc.lagType == lagTypeLACP {
  lacp.LacpMode = telemetry.Lacp_LacpActivityType_ACTIVE
} else {
  lacp.LacpMode = telemetry.Lacp_LacpActivityType_UNSET
}
lacpPath := d.Lacp().Interface(tc.aggID)
fptest.LogYgot(t, "LACP", lacpPath, lacp)
lacpPath.Replace(t, lacp)
```

Reply:
Done and latest push has merge LACP and STATIC